hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/ServerCall.java

```diff
@@ -431,31 +436,42 @@ public boolean isClientCellBlockSupported() {
 
   @Override
   public long getResponseCellSize() {
-    return responseCellSize;
+    return responseCellSize.get();
   }
 
   @Override
-  public void incrementResponseCellSize(long cellSize) {
-    responseCellSize += cellSize;
+  public long getNumsOfGet() {
+    return getsNum.get();
+  }
+
+  public long incrementGetsNum() {
+    return this.getsNum.incrementAndGet();
   }
 
   @Override
-  public long getResponseBlockSize() {
-    return responseBlockSize;
+  public void addResultSize(Result result) {
+    if (result != null && !result.isEmpty()) {
+      for (Cell c : result.rawCells()) {
+        this.responseCellSize.addAndGet(PrivateCellUtil.estimatedSerializedSizeOf(c));
```

Comment:
Seems the addSize() in RSRpcServices was also trying to do some calculation based on type of cell. That is not needed any more?